pulp_rpm/app/models.py

```diff
@@ -195,14 +195,14 @@ class Meta:
         )
 
 
-class SrpmContent(RpmContent):
+class SourcePackage(Package):
     """
-    The "srpm" content type.
+    The "SourcePackage" content type. Formerly "srpm" in Pulp 2.
 
-    (same as the "rpm" content type)
+    (same as the "package" content type)
     """
 
-    TYPE = 'srpm'
+    TYPE = 'source'
```

Comment:
i am not sure about calling srpm just source.
Also do we plan to update other references like RpmPublisherSerializer?

Reply:
Agreed on not calling it 'source'. Maybe 'source-package' or 'srpm'?

I was not planning on updating RpmPublisher/RpmRemote/etc as I inferred that 'rpm' referrs to the plugin and not a content type (ie a remote apples to all content types (packages, errata, etc) and the same with a publisher).